CruCentralCoast/Events/EventDetailsVC.swift

```diff
@@ -33,11 +33,12 @@ class EventDetailsVC: UIViewController {
         
         guard let street = self.event?.location?.street,
```

Comment:
Currently, none of the attributes of a location are required. And not every event has them (some are TBD). With your guard let, the viewDidLoad method would return immediately, preventing any of the rest of the UI elements from being set up properly. Instead, just use a normal if let, or use ?? to unwrap it and provide a default value

Reply:
ok that makes sense. lmk if what I did now is the right way to do it.